glide.yaml

```diff
@@ -8,99 +8,109 @@ import:
 - package: k8s.io/kubelet
   repo:    https://github.com/openshift/kubernetes-kubelet.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kube-proxy
   repo:    https://github.com/openshift/kubernetes-kube-proxy.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kube-scheduler
   repo:    https://github.com/openshift/kubernetes-kube-scheduler.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/legacy-cloud-providers
   repo:    https://github.com/openshift/kubernetes-legacy-cloud-providers.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/metrics
   repo:    https://github.com/openshift/kubernetes-metrics.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/sample-apiserver
   repo:    https://github.com/openshift/kubernetes-sample-apiserver.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/sample-cli-plugin
   repo:    https://github.com/openshift/kubernetes-sample-cli-plugin.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/sample-controller
   repo:    https://github.com/openshift/kubernetes-sample-controller.git
-  version: origin-4.4-kubernetes-1.16.2
+  version: origin-4.4-kubernetes-1.17.0-rc.2
 - package: k8s.io/kubernetes
-  repo:    https://github.com/openshift/kubernetes.git
-  version: origin-4.4-kubernetes-1.16.2
-# this matches the 1.16 branch from kube
+  repo:    https://github.com/p0lyn0mial/kubernetes.git
+  version: origin-4.4-kubernetes-1.17.0-rc.2
+- package: k8s.io/cri-api
+  repo:    https://github.com/openshift/kubernetes-cri-api
+  version: origin-4.4-kubernetes-1.17.0-rc.2
+- package: k8s.io/node-api
+  repo:    https://github.com/openshift/kubernetes-node-api
+  version: origin-4.4-kubernetes-1.17.0-rc.2
+# this matches the 1.17 branch from kube
 - package: k8s.io/gengo
   version: 26a664648505d962332bda642b27306bc10d1082
 - package: k8s.io/utils
-  version: 581e00157fb1a0435d4fac54a52d1ca1e481d60e
+  version: e782cd3c129fc98ee807f3c889c0f26eb7c9daf5
 - package: k8s.io/kube-openapi
-  version: 743ec37842bffe49dd4221d9026f30fb1d5adbc4
+  version: 30be4d16710ac61bce31eb28a01054596fe6a9f1
 - package: k8s.io/klog
-  version: v0.4.0
+  version: v1.0.0
 - package: gopkg.in/yaml.v2
   version: v2.2.4
+- package: sigs.k8s.io/structured-merge-diff
+  version: b1b620dd3f06c352be11cbf3dda1a04606bbf10a
+
 # for gengo
 - package: golang.org/x/tools/imports
   version: 6e04913cbbac05c5c0c9d8f237717e292f700e76
 
 # pin for etcd
 - package: google.golang.org/grpc
-  version: v1.23.0
+  version: v1.23.1
 
-# etcd v3.3.17 carry until 4.4
 - package: github.com/coreos/etcd
-  version: v3.3.17
+  version: v3.3.10
```

Comment:
this backleveling isn't correct.  Get up with @hexfusion to find the right level, but at the very least don't backlevel it.

Reply:
this is exactly what upstream uses https://github.com/kubernetes/kubernetes/blob/v1.17.0-rc.2/go.mod#L231

also `github.com/coreos/etcd` was replaced by `go.etcd.io/etcd`

Reply:
I remember some issues in that area I can try to delete it.

Reply:
@hexfusion is away until `1/2` we need to make a decision.

Reply:
we decided to revert it back to `v3.3.17`

Reply:
thanks 100% agree